pyanaconda/ui/tui/spokes/network.py

```diff
@@ -213,13 +230,13 @@ def input(self, args, key):
 
     def apply(self):
         " Apply all of our settings."""
-        self._update_network_data()
+        self._update_network_data(self.supported_devices)
```

Comment:
We need to update network data not only for self.supported_devices (ie devices for which we support text ui configuration) but also for other devices configured by other means eg via boot options or kickstart (bond, vlan, ...). _update_network_data actually always recreates the data.network.network from scratch (although the name of the function doesn't suggest so :/). So in the end we don't seem to need to parametrize the function with devices argument.